build/cygnus.js

```diff
@@ -1,29 +1,28 @@
-var _this2 = this,
-    _arguments = arguments;
-
 /* global cygnus, Blob, Worker, XMLHttpRequest */
 
+const _arguments = arguments;
```

Comment:
Pretty sure this is something to do with the default conversion. I highly doubt this will be necessary to operation.